sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineImpl.java

```diff
@@ -1993,6 +1994,11 @@ public String userData() {
         return this.innerModel().userData();
     }
 
+    @Override
+    public String capacityReservationGroupId() {
+        return this.innerModel().capacityReservation().capacityReservationGroup().id();
```

Comment:
null check

Reply:
Fixed in the [commit#3a75a4b0](https://github.com/Azure/azure-sdk-for-java/pull/42914/commits/3a75a4b0ff711ceaf70d5b6dd3f8c77b69b322ad)